Services/AppointmentAPI/AppointmentAPI.Application/Commands/Appointment/ChangeStatus/ChangeAppointmentsStatusCommand.cs

```diff
@@ -0,0 +1,8 @@
+﻿using AppointmentAPI.Application.Abstractions;
+using AppointmentAPI.Application.Requests.Appointment;
+using MediatR;
+
+namespace AppointmentAPI.Application.Commands.Appointment.ChangeStatus;
+
+public sealed record ChangeAppointmentsStatusCommand(
+    ChangeAppointmentsStatusRequest Request) : BaseRequest<ChangeAppointmentsStatusRequest, Unit>(Request);
```

Comment:
public sealed record ChangeAppointmentsStatusCommand(
    ChangeAppointmentsStatusRequest Request) : BaseRequest<ChangeAppointmentsStatusRequest, Unit>(Request);
```suggestion
public sealed record ChangeAppointmentsStatusCommand(
    ChangeAppointmentsStatusRequest Request
) : BaseRequest<ChangeAppointmentsStatusRequest, Unit>(Request);
```

Reply:
Fixed